README.adoc

```diff
@@ -16,8 +16,8 @@
 :page-permalink: /guides/{projectid}
 :page-related-guides: ['rest-intro', 'microprofile-health', 'cdi-intro']
 :common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/master
-:page-seo-title: Providing metrics from a microservice with MicroProfile Metrics
-:page-seo-description: A tutorial and an example on how to provide metrics from a microservice using MicroProfile Metrics.
+:page-seo-title: Monitoring the metrics of a Java microservice using Eclipse MicroProfile Metrics
+:page-seo-description: A tutorial on how to expose both system and custom application metrics from a Java microservice and monitor endpoints using Eclipse MicroProfile Metrics.
```

Comment:
```suggestion
:page-seo-description: A getting started tutorial and an example on how to expose system and application metrics from a Java microservice and monitor metrics endpoints using Eclipse MicroProfile Metrics.
```

Reply:
We can consider changing from "a Java microservice" to "Java microservices" too.